docs/embedding/sdk/authentication.md

```diff
@@ -10,6 +10,93 @@ title: Embedded analytics SDK - authentication
 
 Notes on handling authentication when working with the SDK.
 
+## Authenticating people from your server
+
+The SDK requires an endpoint in your app's backend that will sign someone into your Metabase and return a token. The SDK will use that token to authenticate calls to Metabase.
+
+The SDK will call this endpoint to get a new token, or to refresh an existing token that's about to expire.
+
+## Example code for generating a token
+
+This example sets up an endpoint in an app, `/sso/metabase`, that creates a token using the shared secret to authenticate calls to Metabase.
+
+```typescript
+const express = require("express");
+
+const jwt = require("jsonwebtoken");
+const fetch = require("node-fetch");
```

Comment:
we should add more imports here, otherwise this code snippet would error out because it's missing imports for `session(...)` and `cors(...)` below.

```suggestion
const express = require("express");
const cors = require("cors");
const session = require("express-session")
const jwt = require("jsonwebtoken");
const fetch = require("node-fetch");
```